contents/events/desir_winterschool2019/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Shaping new approaches to data management in arts and humanities
+slug: ws2019
+type: event
+date: 2019-12-13
+categories:
+  - events
+abstract:
+  The DESIR Winter School provided a unique opportunity to learn about how to
+  maximize the potential of scholarly resources and to take practical steps in
+  opening up research in ethically and legally responsible ways.
+logo: images/dariah-eu--white.png
+featuredImage: images/background.jpg
+tags:
+  - open-science
+  - open-access
+  - data-management
+license: CCBY 4.0
+partners:
+  - dariah-eu
+  - desir
+  - fcsh-nova
+social:
+  - name: website
+    url: http://dariah.eu/
+  - name: email
+    url:
+  - name: flickr
+    url: https://www.flickr.com/photos/151019225@N04/sets/72157679126801606/
```

Comment:
this link seems to point to the 2016 winterschool photos

Reply:
Yes, I have asked our communication officer to upload the photos on Flickr. Once ready, I will change the link accordingly.